### ISSUE TITLE
Map Final Polishing

### ISSUE THREAD

**Issue by SamBehm** [UQdeco2800/2022-studio-1#318]
# **Description**
The map is almost completely implemented in the game, however, there are a few features that must be implemented to improve game-play.

### User Testing

# **Milestones**

**Completion Deadline:** Oct. 12

# **Extras (optional)**

# **Documentation**

# **Members**
- Sam (@SamBehm) (sammmbehm)

**Comment by alexyun0429** [comment]
Hi Sam, just to clarify which team is taking over this?
The previous team 3 with new team 12,13?

**Comment by SamBehm** [comment]
> Hi Sam, just to clarify which team is taking over this? The previous team 3 with new team 12,13?

Yep! Team 13 will be completing this feature, if there's any need for design elements to be implemented, we'll discuss with Team 12!
